img.py

```diff
@@ -30,8 +37,14 @@ def __init__(self):
 		self.pt0 = 0,0
 		self.pt1 = 0,0
 		self.pts = []
-		self.imgW = 1200
-		self.imgH = 800
+		self.imgW = imgW
+		self.imgH = imgH

```

Comment:
Maybe we should make a module that calculates sizes for us and set it. But I'm just being nit picky.